ipam/client/backends/phpipam.py

```diff
@@ -79,6 +80,10 @@ def __init__(self, params):
         self.cur = self.db.cursor()
         self.set_section_id_by_name(section_name)
 
+        if self._get_version() < 1.32:
```

Comment:
The current version of phpipam is 1.3.2 (and not 1.32), is it stored liked that in the DB? :thinking:

Reply:
Yes, see https://github.com/phpipam/phpipam/commit/6c99d5e267f83db8428ca10aae2acc9af547204d#diff-547ed95531a0a2a6dd5a95f522020aab

Reply:
OK, bad...